client/my-sites/media-library/content.jsx

```diff
@@ -140,6 +144,20 @@ export default React.createClass( {
 						i18nOptions
 					);
 					break;
+				case MediaValidationErrors.NOT_ENOUGH_SPACE:
+					message = this.translate(
+						'The file could not be uploaded because there is not enough space left.',
```

Comment:
These types of strings (with mismatched placeholders) are a bad practice, and I've meant to get around to fixing the originals.

See: https://github.com/Automattic/eslint-plugin-wpcalypso/blob/master/docs/rules/i18n-mismatched-placeholders.md

(Should be a warning too in ESLint)